stop.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 echo "Shutting down Akeneo service"
-source .env
-export SOURCE_PATH="${WORKING_DIR}"
 docker-compose down
```

Comment:
Is not about to have or not have a script is about having extra things that causes confusion. The question is to keep it simple, stick to the basics, and make the normal path seamless. You can have extra things aside. A good test to measure it is watch others install your project.

> Building a project should be a single trivial operation. You should not have to check many
little pieces out from source code control. You should not need a sequence of arcane commands
or context dependent scripts in order to build the individual elements. You should
not have to search near and far for all the various little extra JARs, XML files, and other
artifacts that the system requires. You should be able to check out the system with one simple
command and then issue one other simple command to build it.